lib/bunyan.js

```diff
@@ -1168,9 +1152,9 @@ var errSerializer = Logger.stdSerializers.err = function err(err) {
 };
 
 
-// A JSON stringifier that handles cycles safely.
-// Usage: JSON.stringify(obj, safeCycles())
-function safeCycles() {
+// A JSON stringifier that handles cycles safely - tracks seen vals in an Array.
+// Note: Has performance problems when dealing with large objects - issue#445.
```

Comment:
Let's use trentm/node-bunyan#445 instead of "issue#445".  Perhaps note in this comment that this one is the only one that works for node 0.10 (which doesn't have Set).